shared/wallets/create-account/index.js

```diff
@@ -4,20 +4,20 @@ import * as Kb from '../../common-adapters'
 import type {ValidationState} from '../../constants/types/wallets'
 import {EnterName} from '../link-existing/index'
 
-type Props = {
+type Props = {|
   createNewAccountError: string,
   error: string,
   name: string,
   nameValidationState: ValidationState,
-  onBack: () => void,
+  onBack?: () => void,
```

Comment:
`onBack` isn't passed in the container at all currently, so I don't think it really makes sense to have it be a required prop. From the mockups, sometimes the back button is there and sometimes it isn't, depending on how it's opened.